polygeist/include/mlir/Dialect/Polygeist/Transforms/Passes.td

```diff
@@ -90,6 +96,11 @@ def ParallelLower : Pass<"parallel-lower", "mlir::ModuleOp"> {
       ["memref::MemRefDialect", "polygeist::PolygeistDialect",
        "LLVM::LLVMDialect", "scf::SCFDialect"];
   let constructor = "mlir::polygeist::createParallelLowerPass()";
+  let options = [
```

Comment:
We might want to have a `std::unique_ptr<Pass> createParallelLowerPass(const ParallelLowerOptions &)` function declared in `Passes.h` and defined in `ParallelLower.cpp`

Reply:
Good point, added.